saleor/checkout/utils.py

```diff
@@ -156,31 +155,9 @@ def add_variants_to_checkout(
     If quantity is set to 0, checkout line will be deleted.
     Otherwise, quantity will be added or replaced (if replace argument is True).
     """
-    # check quantities
     country_code = checkout.get_country()
-    if not skip_stock_check:
-        check_stock_and_preorder_quantity_bulk(
-            variants,
-            country_code,
-            quantities,
-            channel_slug,
-            check_reservations=bool(reservation_length),
-        )
-
-    channel_listings = product_models.ProductChannelListing.objects.filter(
-        channel_id=checkout.channel.id,
-        product_id__in=[v.product_id for v in variants],
-    )
-    channel_listings_by_product_id = {cl.product_id: cl for cl in channel_listings}
-
-    # check if variants are published
-    for variant in variants:
-        product_channel_listing = channel_listings_by_product_id[variant.product_id]
-        if not product_channel_listing or not product_channel_listing.is_published:
-            raise ProductNotPublished()
-
     checkout_lines = checkout.lines.select_related("variant")
-    variant_ids_in_lines = {line.variant_id: line for line in checkout_lines}
+    variant_ids_in_lines = {line.variant_id: line for line in checkout.lines.all()}
```

Comment:
Why changing checkout_lines to `checkout.lines.all()` here?

Reply:
I made mistake when rebasing. I will change that.